src/main/java/edu/harvard/iq/dataverse/DataverseFeaturedItem.java

```diff
@@ -0,0 +1,87 @@
+package edu.harvard.iq.dataverse;
+
+import edu.harvard.iq.dataverse.util.SystemConfig;
+import jakarta.persistence.*;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.Size;
+
+@NamedQueries({
+        @NamedQuery(name = "DataverseFeaturedItem.deleteById",
+                query = "DELETE FROM DataverseFeaturedItem item WHERE item.id=:id"),
+        @NamedQuery(name = "DataverseFeaturedItem.findByDataverseOrderedByDisplayOrder",
+                query = "SELECT item FROM DataverseFeaturedItem item WHERE item.dataverse = :dataverse ORDER BY item.displayOrder ASC")
+})
+@Entity
+@Table(indexes = @Index(columnList = "displayOrder"))
+public class DataverseFeaturedItem {
```

Comment:
We don't need a SQL upgrade script because we're creating a new table, is that right?